src/collection/components/CollectionOrBundleOverview.tsx

```diff
@@ -501,9 +504,44 @@ const CollectionOrBundleOverview: FunctionComponent<CollectionOrBundleOverviewPr
 		);
 	};
 
-	const renderCell = (collection: Avo.Collection.Collection, colKey: string) => {
+	const renderCell = (collection: Collection, colKey: string) => {
 		const { id } = collection;
 
+		const sharedWithNames = collection.contributors.map((contributor) => {
+			const fullName = contributor.profile?.usersByuserId?.full_name;
+			const orgName = contributor.profile?.organisation?.name;
+
+			if (contributor.profile?.organisation?.name) {
+				return `${fullName} (${orgName}) `;
+			} else {
+				return `${fullName} `;
+			}
+		});
+
+		const shareTypeTitle =
+			collection.share_type === CollectionShareType.GEDEELD_MET_MIJ
+				? tText('collection/views/collection-overview___gedeeld-met-mij')
+				: collection.share_type === CollectionShareType.GEDEELD_MET_ANDERE
+				? tText('collection/views/collection-overview___gedeeld-met-anderen')
+				: tText('collection/views/collection-overview___mijn-collectie');
```

Comment:
OPTIONAL: i sometimes use this notation instead of a double tertiary operator since it is slightly more readable. The same applies to the 2 consts below
```suggestion
		const shareTypeTitle = {
			[CollectionShareType.GEDEELD_MET_MIJ]: tText('collection/views/collection-overview___gedeeld-met-mij'),
			[CollectionShareType.GEDEELD_MET_ANDERE]: tText('collection/views/collection-overview___gedeeld-met-anderen'),
			[CollectionShareType.NIET_GEDEELD]: tText('collection/views/collection-overview___mijn-collectie'),
		}[collection.share_type];
```